src/templates/bespoke/presenter/presenter-view.tsx

```diff
@@ -140,6 +145,33 @@ const presenterView = (deck) => {
     setInterval(update, 250)
   }
 
+  let isDragging = false
+
+  const startDragging = () => (isDragging = true)
+  const endDragging = () => (isDragging = false)
+
+  const onDragging = (event: MouseEvent) => {
+    if (!isDragging) return
+
+    const leftColWidth = event.clientX
+
+    const dragbarWidth = parseInt(
+      getComputedStyle($(classes.dragbar)).width.replace('px', '')
+    )
+
+    const cols = [
+      leftColWidth,
+      dragbarWidth,
+      $(classes.container).clientWidth - 2 * dragbarWidth - leftColWidth,
+    ]
+
+    const newColDefn = cols.map((c) => c.toString() + 'px').join(' ')
```

Comment:
The grid no longer follow the change of window size after splitter was dragged.

I recommend to use [CSS custom property](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) and leave resize logic to CSS. The script for dragging also would become super smart like following.

```css
--bespoke-marp-presenter-split-ratio: 66%;

grid-template:
  'current dragbar next' minmax(140px, 1fr)
  'current dragbar note' 2fr
  'info    dragbar note' 3em /
  var(--bespoke-marp-presenter-split-ratio) 0px 1fr;
```

```typescript
const onDragging = (event: MouseEvent) => {
  if (!isDragging) return

  $(classes.container).style.setProperty(
    '--bespoke-marp-presenter-split-ratio',
    `${(event.clientX / document.documentElement.clientWidth) * 100}%`
  )

  event.preventDefault()
}
```